spec/rails_helper.rb

```diff
@@ -1,6 +1,8 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 require 'simplecov'
 require 'simplecov-json'
+require 'webmock/rspec'
+WebMock.disable_net_connect!(allow_localhost: true)
```

Comment:
What does this do?

Reply:
This prevents actually making a request to the real oscn website while we're testing
It forces us to mock up the responses or it throws an erroe